client/src/containers/app.coffee

```diff
@@ -17,10 +18,11 @@ App = React.createClass
     loginActions = bindActionCreators LoginActions, dispatch
     pageLocationActions = bindActionCreators PageLocationActions, dispatch
     repoActions = bindActionCreators RepoActions, dispatch
+    registerActions = bindActionCreators RegisterActions, dispatch
 
     div {},
       Header {login, loginActions, pageLocationActions}
-      MainSection {login, repo, repoActions, pageLocation, loginActions, pageLocationActions}
+      MainSection {login, repo, repoActions, pageLocation, loginActions, pageLocationActions, registerActions}
```

Comment:
I just reviewed the docs: http://rackt.github.io/redux/docs/api/bindActionCreators.html
Only components that call `connect` will get `dispatch` through props. At least that's what I think.